tsconfig.json

```diff
@@ -3,7 +3,8 @@
     "module": "commonjs",
     "target": "es6",
     "lib": ["es6"],
-    "sourceMap": true
+    "sourceMap": true,
+    "newLine":  "lf"
```

Comment:
When rebuilding the project locally, CRLF is used. That's a bit painful to maintain.

Docs: https://www.typescriptlang.org/tsconfig#newLine